### PR TITLE
[doc] devtools: Setup ots git integration

### PR DESCRIPTION
**Pull request by maflcko**
Document the simple steps on how to set up ots git integration.

### CODE REVIEW

File: contrib/devtools/README.md [hunk 1]
```diff
@@ -144,6 +144,13 @@ Configuring the github-merge tool for the bitcoin repository is done in the foll
     git config githubmerge.testcmd "make -j4 check" (adapt to whatever you want to use for testing)
     git config --global user.signingkey mykeyid (if you want to GPG sign)
 
+Create and verify timestamps of merge commits
+---------------------------------------------
+To create or verify timestamps on the merge commits, download the OpenTimestamps
```
Comment (conscott) [inline contrib/devtools/README.md]: Maybe link to OTS releases?

**Review by conscott — COMMENTED** [review]
Concept ACK

**Comment by laanwj** [comment]
ACK

File: contrib/devtools/README.md (hunk 1 above)
Comment (petertodd) [reply contrib/devtools/README.md]: Yeah, this instruction is a bit misleading: it's not just enough to download that script, you also have to install OpenTimestamps. The easiest way to do that is simply run:

    pip3 install opentimestamps-client

Comment (MarcoFalke) [reply contrib/devtools/README.md]: I'd rather link to the "manual" installation, since it seems common to typosquat package names for pip.

**Comment by practicalswift** [comment]
Concept ACK

File: contrib/devtools/README.md (hunk 1 above)
Comment (petertodd) [reply contrib/devtools/README.md]: I don't think that's much of a risk, given that we'd be providing copy-n-pastable instructions right there.

**Comment by MarcoFalke** [comment]
Tried to address feedback

**Comment by MarcoFalke** [comment]
@sipa @jonasschnelli This might be of interest to you

**Comment by laanwj** [comment]
re-ACK fa385c3a8be0889d4c50f5168e8de0c6edee3597

**Pull request merged**